components/Slider/src/MDCSlider.h

```diff
@@ -172,6 +194,7 @@ IB_DESIGNABLE
  The radius of the cursor (thumb).
 
  Default value is 6 points.
+ @note Has no effect if @c statefulAPIEnabled is @c YES.
```

Comment:
Can we discourage clients from using this? I hope we are planning to deprecate this property in the future. 

Would you be willing to follow the pattern adopted for `trackBackgroundColor`, `color`, and others and document the upcoming deprecation as well as the recommended alternative (the new `thumbRadius:forState:` APIs)?

Reply:
Done